utilities/unittests/test_oadp.py

```
@@ -589,3 +606,222 @@ def test_create_rhel_vm_running_vm_exception(
+
+        # Parent __exit__ should still be called
+        mock_restore_exit.assert_called_once_with(None, None, None)
+
+    @patch("utilities.oadp.unique_name")
+    @patch("utilities.oadp.Restore.__init__")
+    @patch("utilities.oadp.Restore.__exit__")
+    @patch("utilities.oadp.delete_velero_resource")
+    @patch("utilities.oadp.LOGGER")
+    def test_velero_restore_exit_delete_exception(
+        self,
+        mock_logger,
```

Ruff (0.14.10)

Unused method argument: `mock_logger`

(ARG002)